pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1793,14 +1794,29 @@ func (dm *DockerManager) runContainerInPod(pod *v1.Pod, container *v1.Container,
 		return kubecontainer.ContainerID{}, fmt.Errorf("InspectContainer: %v", err)
 	}
 
-	// Create a symbolic link to the Docker container log file using a name which captures the
-	// full pod name, the container name and the Docker container ID. Cluster level logging will
-	// capture these symbolic filenames which can be used for search terms in Elasticsearch or for
-	// labels for Cloud Logging.
+	dockerLoggingDriver := ""
+	dockerInfo, err := dm.client.Info()
+	if err != nil {
+		glog.Errorf("Failed to execute Info() call to the Docker client: %v", err)
+	} else {
+		dockerLoggingDriver = dockerInfo.LoggingDriver
+		glog.V(10).Infof("Docker logging driver is %s", dockerLoggingDriver)
+	}
+
 	containerLogFile := containerInfo.LogPath
-	symlinkFile := LogSymlink(dm.containerLogsDir, kubecontainer.GetPodFullName(pod), container.Name, id.ID)
-	if err = dm.os.Symlink(containerLogFile, symlinkFile); err != nil {
-		glog.Errorf("Failed to create symbolic link to the log file of pod %q container %q: %v", format.Pod(pod), container.Name, err)
+	if containerLogFile != "" {
```

Comment:
Actually, we should move `client.Info` inside else, too.

However, because we are going to deprecate dockertools, I'm fine to keep it is now.